helm/ocs-monkey-generator/values.yaml

```diff
@@ -15,6 +15,23 @@ osio:
   kernelUntar: 6
   kernelRm: 6
 
+# Set this parameter to install chaos runner
+runChaosRunner: false
+
+# Name of chaos runner deployment
+chaosName: ocs-monkey-chaos
```

Comment:
This should probably be removed and everywhere it's referenced in the templates, and auto-generated name like `'{{ template "ocs-monkey-generator.fullname" . }}-chaos'` should be used instead.
Having a hardcoded resource name is a source of conflicts (at deployment time) in helm charts.